### PR TITLE
Update subctl verify messages

### PR DESCRIPTION
**Pull request by tpantelis**
The help message referred to incorrect flags. Also expanded/improved the help
and prompt messages.

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

### CODE REVIEW

File: pkg/subctl/cmd/verify.go [hunk 1]
```diff
@@ -64,10 +64,17 @@ func addVerifyFlags(cmd *cobra.Command) {
...
+is functioning properly. The verifications performed are controlled by the --only and --enable-disruptive
+flags. All verifications listed in --only are performed with special handling for those deemed as disruptive.
+A disruptive verification is one that changes the state of the clusters as a side effect. If running the
+command interactively, you will be prompted for confirmation to perform disruptive verifications unless
+the --enable-disruptive flag is also specified. If running non-interactively (that is with no stdin),
+--enable-disruptive must be specified otherwise disruptive verifications are skipped.
+
+The following verifications are deemed disruptive:
+
+    gateway-failover`,
```
Comment (skitt) [inline pkg/subctl/cmd/verify.go:77]: Could this be constructed from `verifyE2EDisruptivePatterns`?

Comment (tpantelis) [reply pkg/subctl/cmd/verify.go:77]: yeah good point.

**Comment by tpantelis** [comment]
> I think the `go.mod` and `go.sum` changes should be isolated...

yeah technically but seemed easiest to just include here.  I'll remove from this PR.

File: pkg/subctl/cmd/verify.go (hunk 1 above)
Comment (tpantelis) [reply pkg/subctl/cmd/verify.go]: Done

**Pull request merged**